azure-automation-plus-dsc-lab/dsc/xActiveDirectory/Assert-HADC.ps1

```diff
@@ -0,0 +1,91 @@
+[System.Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidUsingComputerNameHardcoded', '')]
+[System.Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidUsingConvertToSecureStringWithPlainText', '')]
+param()
+
+# A configuration to Create High Availability Domain Controller
+$secpasswd = ConvertTo-SecureString "Adrumble@6" -AsPlainText -Force
```

Comment:
remove hardcoded password - this will need to be passed in

Reply:
@bmoore-msft : I didn't author this xActiveDirectoryModule or it's corresponding Assert-HADC.ps1 script. This credential isn't used as the domain administrator or safemode administrator password anyway. It's not hard coded after-all because it is specified in the azuredeploy.parameters.json file as:
    "adminPassword": {
      "value": "GEN-PASSWORD"
    },